src/server/chunk_server/file_chunk_manager.cc

```diff
@@ -102,7 +102,7 @@ google::protobuf::util::StatusOr<std::string> FileChunkManager::ReadFromChunk(
   auto file_chunk = result.ValueOrDie();
 
   // Check that we aren't trying to read data that isn't there
-  if (start_offset >= file_chunk->data().length()) {
+  if (start_offset > file_chunk->data().length()) {
```

Comment:
Why eliminating the equal sign here? an offset = data().length() is out of bound right?